src/components/data-management/SamplesTable.jsx

```diff
@@ -48,6 +50,18 @@ const exampleDatasets = [
   },
 ];
 
+// If a sample exists in projects but not in the samples file,
+// there is an error uploading the sample. We should show an error for that sample.
+const sampleUploadErrorTemplate = {
+  ...sampleTemplate,
+  name: 'UPLOAD ERROR: Please reupload sample',
+  sampleFiles: {
```

Comment:
why adding these under `sampleFiles`, when it looks like code will be reading them from `files`?

Reply:
Yes, it should be under files, my bad. I'll change it.